test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go

```diff
@@ -566,9 +566,75 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 			By("Saving the old performance profile")
 			initialProfile = profile.DeepCopy()
 		})
+		When("RealTime Workload with RealTime Kernel set to false", func() {
+			It("[test_id:50991][crit:high][vendor:cnf-qe@redhat.com][level:acceptance]should update kernel arguments and tuned accordingly", func() {
+				By("Modifying profile")
+				profile.Spec.WorkloadHints = &performancev2.WorkloadHints{
+					HighPowerConsumption: pointer.Bool(false),
+					RealTime:             pointer.Bool(true),
+				}
+
+				profile.Spec.RealTimeKernel = &performancev2.RealTimeKernel{
```

Comment:
we are checking workloadHints, better not to add noise. If I am not missing anything its not necessary to check in this tests the value of rt kernel.

Reply:
No but in this test case we are checking workload hints with Realtime kernel disabled explicitly.

Reply:
yes, but why? there is any different between workload Hints HighPower to false and realtime to true and realtime Kernel disabled explicit or even enabled?